### DISCUSSION TITLE
How to transfer yolo *.txt （the new segment） to *.json（labelme labeled segment）?

### DISCUSSION THREAD

**Discussion by KroitAax** [wkentaro/labelme#1320] [Ideas / Feature requests]
How to transfer yolo *.txt （the new segment） to *.json（labelme labeled segment）?I hope everyone can give some tips, thank you.

**Comment by ZJDATY** [comment]
Chatgpt maybe a good choice.

**Comment by Abonia1** [comment]
Hello, 
Recently I was searching for similar functionality.I haven't find anything for segmentation task.

So, I have recently released  open source, [yolosegment2labelme](https://github.com/Abonia1/yolosegment2labelme) is a library designed to transform YOLO segmentation masks into a format compatible with annotation tools such as LabelMe and Anylabeling, simplifying the annotation process for image segmentation tasks.

`pip install yolosegment2labelme`

Hope this helps.
